descriptors/ModuleDescriptor-template.json

```diff
@@ -432,12 +443,26 @@
         "orders-storage.po-line-number.get"
       ]
     },
+    {
+      "permissionName" : "orders-storage.order-lines.collection.get",
+      "displayName" : "order-lines get",
+      "description" : "Get a collection of PO lines"
+    },
+    {
+      "permissionName" : "orders-storage.order-lines.all",
+      "displayName" : "All order lines perms",
+      "description" : "All permissions for the Order lines",
+      "subPermissions" : [
+        "orders-storage.order-lines.collection.get"
+      ]
+    },
     {
       "permissionName" : "orders-storage.module.all",
       "displayName" : "All orders-storage-module perms",
       "description" : "All permissions for the orders-storage module",
       "subPermissions" : [
         "orders-storage.alerts.all",
+        "orders-storage.order-lines.collection.get",
```

Comment:
Probably `orders-storage.order-lines.all` should be used here...

Reply:
Not required anymore